addons/product/models/product.py

```diff
@@ -128,6 +128,8 @@ class ProductProduct(models.Model):
         'product.packaging', 'product_id', 'Product Packages',
         help="Gives the different ways to package the same product.")
 
+    priority = fields.Selection(related='product_tmpl_id.priority', readonly=True, store=True)
```

Comment:
I think we can remove this field declaration for `product.product`. we don't need to be store twice the same information on `product.template` and `product.product` tables. 

Functionally: Nothing change because we don't allow to choose product variant as favorite (without favorite the template linked)
Performance: It shouldn't impact performance because the join with the product template is done anyway, for each search on the model due to the _order which contains `name` (also store only on template).